### PR TITLE
improve java trace output readability

### PR DESCRIPTION
**Pull request by apkunpacker**

### CODE REVIEW

File: src/agent/index.js [hunk 1]
```diff
@@ -3004,9 +3004,9 @@ function traceJava (klass, method) {
       if (config.getString('hook.output') === 'json') {
         traceEmit(traceMessage);
       } else {
-        let msg = `[java trace][${traceMessage.timestamp}] ${klass}:${method} - args: ${JSON.stringify(args)}. Return value: ${res.toString()}`;
+        let msg = `[JAVA TRACE][${traceMessage.timestamp}] ${klass}:${method} - args: ${JSON.stringify(args)}. Return value: ${res.toString()}`;
         if (config.getBoolean('hook.backtrace')) {
-          msg += ` backtrace: ${traceMessage.backtrace.toString()}`;
+          msg += ` backtrace: \n${traceMessage.backtrace.toString().split(",").join("\nat ")}\n`;
```
Comment (trufae) [inline src/agent/index.js:3009]:
```suggestion
          msg += ` backtrace: \n${traceMessage.backtrace.toString().split(',').join('\nat ')}\n`;
```

File: plugins/inject-getpid.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        });
+        arm64Writer.putInstruction(0xd2801588);
+        arm64Writer.putInstruction(0xd2800020);
+        arm64Writer.putInstruction(0xd4000001);
+        arm64Writer.putRet();
+        arm64Writer.flush();
+    });
+    const fun = new NativeFunction(getBaseCode, 'int', []);
+    console.log("getPid From syscall : ", fun());
+} else
```
Comment (trufae) [inline plugins/inject-getpid.js:16]: add the missing { } after the else

**Review by trufae — CHANGES_REQUESTED** [review]

File: src/agent/index.js [hunk 3]
```diff
@@ -3004,9 +3004,9 @@ function traceJava (klass, method) {
       if (config.getString('hook.output') === 'json') {
         traceEmit(traceMessage);
       } else {
-        let msg = `[java trace][${traceMessage.timestamp}] ${klass}:${method} - args: ${JSON.stringify(args)}. Return value: ${res.toString()}`;
+        let msg = `[JAVA TRACE][${traceMessage.timestamp}] ${klass}:${method} - args: ${JSON.stringify(args)}. Return value: ${res.toString()}`;
```
Comment (enovella) [inline src/agent/index.js:3007]: If you change this to uppercase, make sure you also change other occurrences of `java trace`

Comment (apkunpacker) [reply src/agent/index.js:3007]: yeah sure

**Review by trufae — COMMENTED** [review]
```js
$ git grep 'java trace'
src/agent/index.js:        let msg = `[java trace][${traceMessage.timestamp}] ${klass}:${method} - args: ${JSON.stringify(args)}. Return value: ${res.toString()}`;
src/agent/index.js:        traceEmit(`[java trace][${timestamp}] Intercept return for ${klass}:${method} with ${value}`);
```

**Comment by trufae** [comment]
lgtm

File: plugins/inject-getpid.js [hunk 4]
```diff
@@ -12,6 +12,12 @@ if (Process.arch == "arm64" && Process.platform == "linux") {
...
     const fun = new NativeFunction(getBaseCode, 'int', []);
-    console.log("getPid From syscall : ", fun());
-} else
-    console.log("Unsupported arch");
\ No newline at end of file
+    console.log('getPid From syscall : ', fun());
+} else {
+    console.log('Unsupported arch');
+}    
+
```
Comment (trufae) [inline plugins/inject-getpid.js:19]: unnecessary empty lines

**Review by trufae — CHANGES_REQUESTED** [review]
just this little thing

**Pull request merged**